### PR TITLE
fix(prepopulate): ignore conflicts in prepopulate

### PR DESCRIPTION
**Pull request by petrjasek**
in case data is there already there is nothing to prepopulate,
so ignore the resource

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: apps/prepopulate/app_prepopulate.py [hunk 1]
```diff
@@ -79,7 +80,10 @@ def prepopulate_data(file_name, default_user=get_default_user()):
                 if not res:
                     raise Exception()
             else:
-                ids = service.post([data])
+                try:
+                    ids = service.post([data])
+                except werkzeug.exceptions.Conflict:
+                    ids = [data.get('_id')]  # data with given id is there already
```
Comment (jerome-poisson) [inline apps/prepopulate/app_prepopulate.py]: why using data.get here? data['_id'] would not be better if we know that _id exists?

Comment (petrjasek) [inline apps/prepopulate/app_prepopulate.py]: refactored this, there were items without id with conflicts, so handling this case now too

**Comment by jerome-poisson** [comment]
:+1:

**Pull request merged**